src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/src/TelemetryReporter.ts

```diff
@@ -31,6 +31,10 @@ export class TelemetryReporter {
         this.reportError('VSCode.Razor.ErrorOnServerStart', error);
     }
 
+    public reportErrorOnServerStop(error: Error) {
+        this.reportError('VSCode.Razor.ErrorOnServerStop', error);
```

Comment:
Does this new event need to be registered externally?

Reply:
We've recently changed our focus on how we do telemetry. For now i'd leave this out because we aren't acting on any of these telemetry events as of late.

For future knowledge for every telemetry event we add we have to go through an approval process for them as well to ensure we can capture the data

Reply:
Thanks for letting me know. I've removed it.